### ISSUE TITLE
Moving to language server

### ISSUE THREAD

**Issue by castwide** [guskovd/emacs-solargraph#4]
The legacy web server (`solargraph server`) is liable to be deprecated in an upcoming version. Clients are recommended to switch to the language server if possible.

Discussion here: https://github.com/castwide/solargraph/issues/50

**Comment by VonFry** [comment]
+1

And using [lsp-mode](https://github.com/emacs-lsp/lsp-mode) for emacs.

**Comment by foreverqihe** [comment]
+1

**Comment by snowman** [comment]
> +1
> 
> And using [lsp-mode](https://github.com/emacs-lsp/lsp-mode) for emacs.

Do you mean do not use `emacs-solargraph` anymore (uninstall if installed), and switch to `[lsp-mode](https://github.com/emacs-lsp/lsp-mode)`? :thinking:

**Comment by guskovd** [comment]
Yes, I think this is the right decision. This library will no longer be developed
